### PR TITLE
Post new service order

### PR DESCRIPTION
**Pull request by enkryptor**

### CODE REVIEW

File: src/app/core/services/orders-api.service.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {Injectable} from '@angular/core';
+import {OrderModel, OrderPostModel} from '@app/core/models/order-model';
+import {ApiClientService} from '@app/core/services/api-client.service';
+import {environment} from '@env/environment';
+import {Observable} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class OrdersApiService {
```
Comment (ennaque) [inline src/app/core/services/orders-api.service.ts:10]: why not to extend abstract api service here

File: src/app/core/services/translation.service.ts [hunk 2]
```diff
@@ -53,24 +61,23 @@ export class TranslationService {
...
         this.getStorage().then(
-            (defaultLang: string | null) => null !== defaultLang ?
-                this.translator.setDefaultLang(defaultLang) :
-                this.setLang(this.LANGUAGES.en)
+            (lang: string | null) => {
+                if (lang) {
+                    this.translator.use(lang);
+                } else {
+                    this.translator.use(this.LANGUAGES.en);
+                    this.setStorage(this.LANGUAGES.en);
```
Comment (ennaque) [inline src/app/core/services/translation.service.ts:78]: can call this.setLang()

File: src/app/service/components/service-details-page/service-details-page.component.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                        <ion-button routerLink="/search" routerDirection="back">
+                            <ion-icon name="arrow-back-outline"></ion-icon>
+                        </ion-button>
+                    </ion-buttons>
+                    <ion-title>{{ 'Информация об услуге' }}</ion-title>
+                </ion-item>
+                <ng-container *ngIf="service">
+                    <app-service-widget [service]="service" [master]="master" [alwaysExpanded]="true"></app-service-widget>
+                    <ion-button [routerLink]="'/order/' + service.id">Выбрать</ion-button>
+                </ng-container>
```
Comment (ennaque) [inline src/app/service/components/service-details-page/service-details-page.component.html:17]: no translations

**Review by ennaque — APPROVED** [review]

File: src/app/service/components/service-details-page/service-details-page.component.html (hunk 3 above)
Comment (enkryptor) [reply src/app/service/components/service-details-page/service-details-page.component.html:17]: service-details-page.component.html is a stub

**Pull request merged**